Library/Homebrew/config.rb

```diff
@@ -50,7 +50,11 @@ def mkpath
 HOMEBREW_TEMP = Pathname.new(ENV.fetch("HOMEBREW_TEMP", "/tmp"))
 
 unless defined? HOMEBREW_LIBRARY_PATH
-  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent.join("Homebrew")
+  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
+end
+
+unless defined? HOMEBREW_CABAL_CONFIG_PATH
```

Comment:
It affects what code paths you go down in Library/Homebrew/language/cabal.config but happy to move them wherever makes the most sense.